config/tests/testgrids/config_test.go

```diff
@@ -353,7 +353,6 @@ var noPresubmitsInTestgridPrefixes = []string{
 	"GoogleCloudPlatform/k8s-multicluster-ingress",
 	"kubeflow/pipelines",
 	"kubernetes-sigs/cluster-capacity",
-	"kubernetes-sigs/gcp-compute-persistent-disk-csi-driver",
```

Comment:
Since you're removing the repo from the ignore list, you may need to make sure all the other existing jobs are setting the testgrid annotations correctly.

Reply:
I am really lost here. I removed this because of a test failure. Now I added annotation in presubmit job, still failing :(

Reply:
I think I understand better now..